build.sh

```diff
@@ -118,6 +118,14 @@ NETBOX_PATH="${NETBOX_PATH-.netbox}"
 # Fetching the NetBox source
 ###
 if [ "${2}" != "--push-only" ] && [ -z "${SKIP_GIT}" ]; then
+  REMOTE_EXISTS=$(git ls-remote --heads --tags "${URL}" "${NETBOX_BRANCH}" | wc -l)
+  if [ "${REMOTE_EXISTS}" != "1" ]; then
```

Comment:
Shouldn't it better be the check for no remotes?

```suggestion
  if [ "${REMOTE_EXISTS}" == "0" ]; then
```